site/src/pages/docs/advanced-structured-logging.mdx

```diff
@@ -79,6 +79,94 @@ retrieving the information collected during request life cycle in a machine-frie
 | `responseContentPreview`              | the preview of the response content                                  |
 +---------------------------------------+----------------------------------------------------------------------+
 
+The duration of `totalDurationNanos` is different in the client-side and the server-side.
+
+For the server-side, `totalDurationNanos` starts when the server receives the <type://RequestHeaders> and
+ends when the server sends the response fully:
+
+```bob-svg
+
+           +---------------------------------------------------------------------+
+           |<--------------------------totalDurationNanos----------------------->|
+           +----------------------------+----------------------------------------+
+           |<---requestDurationNanos--->|
+           +----------------------------+            +---------------------------+
+           ^                            ^            |<--responseDurationNanos-->|
+           :                            :            +---------------------------+
+           :                            :            ^                           ^
+  Received RequestHeaders    Received the last frame :                           :
+                                                     :                           :
+                                           Sending ResponseHeaders    Sending the last frame
+```
+
+As you noticed, `totalDurationNanos` is not the sum of `requestDurationNanos` and `responseDurationNanos`.
+The sum of `requestDurationNanos` and `responseDurationNanos` even can be greater than `totalDurationNanos` when
+the <type://ResponseHeaders> is sent before the service receives the <type://HttpRequest> fully:
+
+```bob-svg
+
+           +---------------------------------------------------------------------+
+           |<--------------------------totalDurationNanos----------------------->|
+           +--------------------------------------------+------------------------+
+           |<-----------requestDurationNanos----------->|
+           +-----------------------+--------------------+------------------------+
+                                   |<-----------responseDurationNanos----------->|
+                                   +---------------------------------------------+
+                                   ^
+                                   :
+             Sending ResponseHeaders before the service fully receives the request
+```
+
+On the other hand, `totalDurationNanos` of the client-side contains
+[connectionAcquisitionDuration](/docs/advanced-structured-logging#client-connection-timing-properties):
```

Comment:
..., because a client may have to establish a new connection or acquire an existing connection from the connection pool before sending a request:

Reply:
That's much better. Fixed. 😄